PocketKit/Sources/PocketKit/Recommendation/Recommendation+Extenstions.swift

```diff
@@ -11,7 +11,11 @@ extension Recommendation {
     }
 
     var bestDomain: String? {
-        item.syndicatedArticle?.publisherName ?? item.domainMetadata?.name ?? item.domain ?? item.bestURL.host
+        item.syndicatedArticle?.publisherName
+        ?? item.domainMetadata?.name
+        ?? item.domain
+        ?? URL(string: item.bestURL)?.host
+        ?? "" // TODO: What should be the final fallback string?
```

Comment:
what about item.bestURL itself?

Reply:
Sounds reasonable to me! I'll make the change in the SavedItem-related PR